README.md

```diff
@@ -15,13 +15,13 @@ cluster admins or by other automated tools.
 The GitOps Toolkit components interact with each other via Kubernetes
 events and are responsible for the reconciliation of their designated API objects.
 
-## `gotk` installation
+## `flux` installation
 
 With Homebrew:
 
 ```sh
 brew tap fluxcd/tap
-brew install gotk
+brew install flux
```

Comment:
This is not going to work, there is a flux formula on brew.

Reply:
I think this should be `brew install fluxcd/tap/flux`

Reply:
Other option is to rename it to `flux-cli`, that would also work if we ever move to `homebrew-core`.